nb/api/as.nb

```diff
@@ -1,5 +1,8 @@
 md`
-# shape.as(id)
+### as()
```

Comment:
This is all really feeling like such a complete system 👍

Reply:
Yeah, getting there.